### ISSUE TITLE
Open repository into container: support that the user can enter the credentials

### ISSUE THREAD

**Issue by egamma** [microsoft/vscode-remote-release#1941]
pls see https://github.com/microsoft/vscode-remote-release/issues/1578#issuecomment-560415863

**Comment by Chuxel** [comment]
@ chrmarti Is this also covered the work on #1578 this iteration?

**Comment by chrmarti** [comment]
This tracks the keyboard interaction when required by Git / SSH. This is not covered yet. The current limitation is that the credentials must be available in the Git credentials helper or the SSH agent to avoid the need for keyboard interaction.

**Comment by bhack** [comment]
I have still this problem https://github.com/microsoft/vscode-remote-release/issues/2087#issuecomment-570915095 on linux.

**Comment by bhack** [comment]
On Linux I still need to add in the `Dockerfile` `RUN install -d -m 770 -o vscode -g vscode /home/vscode/.ssh/ && ssh-keyscan -t rsa github.com >> /home/vscode/.ssh/known_hosts`

**Comment by chrmarti** [comment]
If you don't already have a `known_hosts` file in the container, we now copy the one from the host machine over to the container.

**Comment by bhack** [comment]
> we now?

When? I'am using insiders.

**Comment by chrmarti** [comment]
@bhack You're right that (#2285) is still open, I fixed this only for the Repository Container case's `git clone` stage.
